### DISCUSSION TITLE
[Enhancement] [date-picker] el-date-picker范围选择可以只显示一个面板

### DISCUSSION THREAD

**Discussion by izhaorui** [element-plus/element-plus#13020] [Feature Request]
![image](https://github.com/element-plus/element-plus/assets/16451400/83fb56b1-703b-42c7-939e-8457cbdb3a04)
目前el-date-picker 在移动端支持不太友好，希望增加属性可以只显示一个日期范围

**Comment by perweitao** [comment]
需要这个功能，碰到很多次修改起来很不方便
